src/einundzwanzig.py

```diff
@@ -0,0 +1,99 @@
+from telegram import Update
+import requests
+from bs4 import BeautifulSoup
+from telegram.ext.callbackcontext import CallbackContext
+import json
+import config
+import qrcode
+import os
+
+
+# Define podcast formats
+urlAll = '/podcast/'
+urlNews = '/podcast/news/'
+urlLese = '/podcast/lesestunde/'
+urlWeg = '/podcast/der-weg/'
+urlInterviews = '/podcast/interviews/'
+
+def getEpisode(url: str) -> str:
+    """
+    Returns the link to the most recent episode or an error message, if
+    the request fails
+    """
+    try:
+        r = requests.get(config.EINUNDZWANZIG_URL + url, timeout=5)
+        doc = BeautifulSoup(r.text, "html.parser")
+        return f"{config.EINUNDZWANZIG_URL + doc.select('.plain')[0].get('href')}"
+    except:
+        return "Es kann aktuell keine Verbindung zum Server aufgebaut werden. Schau doch solange auf Spotify vorbei: https://open.spotify.com/show/10408JFbE1n8MexfrBv33r"
+
+def episode(update: Update, context: CallbackContext):
+    """
+    Sends a link to the most recent podcast episode
+    """  
+    try:
+        format = str(context.args[0]).lower()
+    except: 
+        format = "alle"
+    
+    if format == "news":
+        message = getEpisode(urlNews)
+    elif format == "lesestunde":
+        message = getEpisode(urlLese)
+    elif format == "alle":
+        message = getEpisode(urlAll)
+    elif format == "weg":
+        message = getEpisode(urlWeg)
+    elif format == "interview":
+        message = getEpisode(urlInterviews)
+    else:
+        message = 'Das ist kein gültiges Podcast-Format! Bitte gibt eins der folgenden Formate an: Alle, Interviews, Lesestunde, News, Weg'
+
+    context.bot.send_message(chat_id=update.effective_chat.id, text=message)
+
+def getInvoice(amt, shoutout):
+    TALLYDATA = {
+    'type': 'fundraiser',
+    'id': 'zfxqtu',
+    'satoshi_amount': '21',
+    'payment_method': 'ln',
+    'message' : ''
```

Comment:
I don't think this parameter exists on the tallycoin api? Can we even send a memo?